packages/kbn-plugin-generator/integration_tests/generate_plugin.test.js

```diff
@@ -61,6 +61,19 @@ describe(`running the plugin-generator via 'node scripts/generate_plugin.js plug
     expect(stats.isDirectory()).toBe(true);
   });
 
+  it(`should create an internationalization config file with a blank line appended to satisfy the parser`, async () => {
+    // Link to the error that happens when the blank line is not there:
+    // https://github.com/elastic/kibana/pull/45044#issuecomment-530092627
+    const intlFile = `${generatedPath}/.i18nrc.json`;
+
+    const stats = await statP(intlFile);
+    expect(stats.isFile()).toBe(true);
+
+    const re = new RegExp(/^\s{1}$/, 'gm');
+    const contents = readFileSync(intlFile, 'utf8');
+    expect(re.test(contents)).toBe(true);
```

Comment:
Feels like maybe we should be explicitly testing that the last line in this file is empty, maybe something like:
```js
const lastLine = readFileSync(intlFile, 'utf8').split('\n').pop()
if (lastLine !== '') {
  throw new Error(`expected last line in ${intlFile} to be empty, received ${JSON.stringify(lastLine)}`);
}
```

Reply:
Cool beans, I'll fixup.  Thanks chief!

Reply:
@spalger you can see in the console that I've manually removed the last line of the file and re-ran the test.  It still passed but should have thrown.

Reply:
Based on the log output the newline is still there

Reply:
If the newline wasn't there `# End` would be on the same line as the last `}`